apps/mail/apps.py

```diff
@@ -0,0 +1,37 @@
+from importlib import import_module
+import inspect
+
+from django.apps import AppConfig, apps as django_apps
+from django.core.checks import register, Warning
+from django.template.loader import get_template
+from django.template.exceptions import TemplateDoesNotExist
+
+
+class MailConfig(AppConfig):
+    name = 'apps.mail'
+
+    def ready(self):
+
+        @register
+        def mailbase_check(app_configs, **kwargs):
```

Comment:
I modelled this after how Django's `check` validates the models, rather than the admin, but I can look into that pattern with the decorators if you think it's worthwhile.

This isn't required to get any of the functionality of the mail app, by the way, it's just so `manage.py check` will warn if any mail classes in `mail.py` are missing their templates.

Reply:
Eh, it's not a major issue, especially if it's significantly more work. I rather get this merged sooner than later so we can stop using SendGrid templates in new projects.

Plus it was just a feeling back in September, the code works and isn't too hard to follow, 👍